docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -15,6 +15,9 @@ Appium automated real device testing supports tests designed to run against a we
 
 See [When to Test on Real Devices](https://docs.saucelabs.com/mobile-apps/supported-devices/#when-to-test-on-real-devices) for deails about real device testing use cases, benefits, and system requirements.
 
+:::caution 
+To use ADB and shell commands, the usage of [vUSB](/mobile-apps/features/virtual-usb) with private devices is necessary as sauce labs does not support ADB without the use of vUSB.
```

Comment:
Can we use the words `Sauce Labs` instead of `sauce labs`. Secondly, ADB can also be used during live testing.
We might also want to mention that we don't support ADB commands for Appium, so being more explicit

Reply:
Corrected.